python/json_util.py

```diff
@@ -0,0 +1,41 @@
+import mgp
+import json
+import urllib.request
+
+
+@mgp.read_proc
+def load_from_path(ctx: mgp.ProcCtx, path: str) -> mgp.Record(objects=mgp.List[object]):
+    """
+    Procedure to load JSON from a local file. 
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being loaded.
+    """
+    with open(path) as json_file:
+        objects = json.load(json_file)
+
+        if type(objects) is dict:
```

Comment:
What happens if objects are of type List?

Reply:
It's fine, they are list and I can easily UNWIND them in Cypher. Tested that on simple ["apple", "banana"] example, since JSON can be array or dict. Dict cannot stay dict, since return type is List and since I want to UNWIND that.

Reply:
That's fine as long as it is written in the documentation